cmake/pkgconf.cmake

```diff
@@ -1,8 +1,8 @@
 # --- generate pkg-config .pc
 set(pc_req_private "")
-if(mpi)
+if(SC_ENABLE_MPI)
   string(APPEND pc_req_private " ompi ompi-c zlib")
```

Comment:
Thanks; unrelated to this PR: are we depending on a particular (OpenMPI) MPI installation?